### PR TITLE
[IMP] if get formview loaded with all use_scrum records load specific

### PR DESCRIPTION
**Pull request by gfcapalbo**
load specific scrum form when fetching use_scrum records.

### CODE REVIEW

File: project_scrum/models/project_task.py [hunk 1]
```diff
@@ -122,3 +122,9 @@ def _read_group_us_id(self, present_ids, domain, **kwargs):
             return user_stories, None
         else:
             return [], None
+
+    @api.multi
+    def get_formview_id(self):
+        if all(self.mapped('use_scrum')):
+            return self.env.ref('project_scrum.view_ps_sprint_task_form2').id
+        return False
```
Comment (hbrunn) [inline project_scrum/models/project_task.py]: better return super here

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Comment by hbrunn** [comment]
codecov is right, a test would be nice. Please rebase afterwards

**Comment by gfcapalbo** [comment]
@hbrunn wrote simple test that covers the two extra code branches. rebased. waiting for CI results.

**Comment by hbrunn** [comment]
tests fail

**Comment by rafaelbn** [comment]
Hi @hbrunn could we merge this?

**Review by rafaelbn — APPROVED** [review]

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
@gfcapalbo please add an extra commit where you update `.travis.yml` with the latest version from MQT to make this mergeable. Squash your commits while being at it

**Comment by rafaelbn** [comment]
Hi @hbrunn @gfcapalbo next week this module will be migrated to v11, it could be great to have this merged :+1:

**Comment by rafaelbn** [comment]
Hi @hbrunn I think all is OK here. Could you retry merge?

**Comment by hbrunn** [comment]
@gfcapalbo please rebase on current 10.0

**Comment by hbrunn** [comment]
actually, we need to update this branch's `.travis.yml` to the latest version from MQT, please add a commit that does that

**Comment by gfcapalbo** [comment]
@hbrunn @rafaelbn added travis from mantainer quality tools , with python 2.7, all green. thank you.

**Comment by rafaelbn** [comment]
Great! @gfcapalbo @hbrunn you have already this module migrated to v11 ready to review :smile:

**Comment by gfcapalbo** [comment]
@rafaelbn thanks

**Pull request merged**